packages/react-select/src/components/Menu.js

```diff
@@ -380,6 +384,7 @@ export const MenuList = (props: MenuListComponentProps) => {
         className
       )}
       ref={innerRef}
+      aria-expanded="true"
```

Comment:
I'm feeling a bit unsure here, but I think `role="listbox"` should go higher up the DOM (the listbox "container" if you will), but since MenuList is conditionally rendered, `aria-expanded="true"` is correct.

Reply:
Listbox should contain options, hence I am putting it with `MenuList`